src/components/Heroes/rankColumns.ts

```diff
@@ -1,90 +1,122 @@
 import { abbreviateNumber, displayHeroId } from '../../utility';
-import { GlobalString } from 'src/types/common/GlobalString'
-import { Heroes } from 'src/types/Hero/Heroes'
-import React from 'react'
+import { GlobalString } from 'src/types/common/GlobalString';
+import { Heroes } from 'src/types/Hero/Heroes';
+import React from 'react';
 
 type Props = {
-  tabType: HeroesTab
-  strings: GlobalString
-}
+  tabType: HeroesTab;
+  strings: GlobalString;
+};
 
 enum HeroesTab {
   PRO = 'pro',
-  PUBLIC = 'public'
+  PUBLIC = 'public',
+  TURBO = 'turbo',
 }
 
 type ProColumn = {
-  displayName: string,
-  field: string,
-  sortFn: boolean,
-  percentBarsWithValue: (row: Row) => string
-}
+  displayName: string;
+  field: string;
+  sortFn: boolean;
+  percentBarsWithValue: (row: Row) => string;
+};
 
 type PublicColumn = {
-  displayName: string,
-  field: string,
-  sortFn: boolean,
-  percentBarsWithValue: (row: Row) => string
-  displayIcon: string
-  colColor: string
-}
+  displayName: string;
+  field: string;
+  sortFn: boolean;
+  percentBarsWithValue: (row: Row) => string;
+  displayIcon: string;
+  colColor: string;
+};
 
 type HeroColumn = {
-  displayName: string,
-  tooltip: string,
-  field: string
-  displayFn: () => React.ReactNode,
-  sortFn: (row: Row) => string
-}
+  displayName: string;
+  tooltip: string;
+  field: string;
+  displayFn: () => React.ReactNode;
+  sortFn: (row: Row) => string;
+};
 
-type PreparedColumn = (ProColumn | PublicColumn | HeroColumn) & ColumnAddition
+type PreparedColumn = (ProColumn | PublicColumn | HeroColumn) & ColumnAddition;
 
 type ColumnAddition = {
-  paddingLeft: number,
-  paddingRight: number,
-  tooltip: string
-}
+  paddingLeft: number;
+  paddingRight: number;
+  tooltip: string;
+};
 
 type Row = {
-  [key in string]: number
-}
+  [key in string]: number;
+};
 
-type Column = ProColumn | PublicColumn | HeroColumn
+type Column = ProColumn | PublicColumn | HeroColumn;
 
 export const rankColumns = (props: Props) => {
-  return props.tabType === HeroesTab.PRO
-    ? generateProTabColumns(props.strings)
-    : generatePublicTabColumns(props.strings);
+  const columns = {
+    [HeroesTab.PRO]: generateProTabColumns(props.strings),
+    [HeroesTab.PUBLIC]: generatePublicTabColumns(props.strings),
+    [HeroesTab.TURBO]: generateTurboTabColumns(props.strings),
+  };
+  return columns[props.tabType];
+};
+
+const generateTurboTabColumns = (strings: GlobalString) => {
+  const heroColumn = generateHeroColumn(strings);
+
+  const combinedColumns = [
+    heroColumn,
+    {
+      displayName: strings.hero_turbo_pick_rate,
+      field: 'pickRateTurbo',
+      sortFn: true,
+      percentBarsWithValue: (row: Row) => decimalToCount(row.turboWinRate, row.turbo_picks),
+    },
+    {
+      displayName: strings.hero_turbo_win_rate,
+      field: 'winRateTurbo',
+      sortFn: true,
+      percentBarsWithValue: (row: Row) => decimalToCount(row.winRateTurbo, row.matchCountTurbo),
```

Comment:
think (pick rate, match count) should be for pick rate, and (winrate, pick count) should be for win rate